assets/pages/oneoff-contributions/components/formFields.jsx

```diff
@@ -53,6 +57,12 @@ function FormFields(props: PropTypes) {
         value={props.postcode || ''}
         onChange={props.postcodeUpdate}
       />
+      <CheckboxInput
+        id="gnm-marketing-preference"
+        value={props.gnmMarketingPreference || false}
```

Comment:
Do you use this field in `CheckboxInput`? Isn't the field `checked`?

Reply:
The name of this field it doesn't matter since it is a prop. Nevertheless, I will change it to `checked`, in order to be more transparent about which attribute will change in the `input` tag.